### PR TITLE
I have updated documentation of part 4 and 3.1

### PR DESCRIPTION
**Pull request by alvarogarinf**
I have updated documentation of part 4 taking into account the commen…ts of the previous deriverable, please review this and feel free to comment anything you want to change

### CODE REVIEW

**Comment by alvarogarinf** [comment]
I have also updated the 3.1 diagram table, because its description was not updated

**Comment by alvarogarinf** [comment]
In point 11 I have added as a technical risk the viade specification

**Review by themrcesi — APPROVED** [review]

**Review by lamasumas — APPROVED** [review]
LGTM

**Review by themrcesi — CHANGES_REQUESTED** [review]
I would remove the MVC pattern since we actually we don´t follow it 100%.

File: src/docs/04_solution_strategy.adoc [hunk 1]
```diff
@@ -3,21 +3,21 @@
...
-| Coveralls | Hosted analysis tool which provide stadistics about our code coverage
-| Code Quality | Write quality code and make it understandable, using the readme for external users understand how the application works and provides information
+| Decision | Purpose | Advantage | Disadvantage
+| ReactJS  | We will use this JavaScript library to build the user interfaces | It makes front-end understandable, easy and fast | At the start of the project, we had no idea nor experience working with React
+| Solid | We have to store the user data in a Solid POD, which will allow us to make the application interoperable and descentralized | The information of each individual will be managed by each individual | Solid is still on development and is not easy to work with
+| Node JS | We will use it for modules management | It offers a wide offer of package via npm | You have to be careful with the packages that you install as the may have vulnerabilities
+| Model-View-Controller | We will use this architectural pattern in order to isolate the presentation layer from the business layer, allowing us to be for flexible | Fast development | MVC architecture is difficult to understand
```
Comment (themrcesi) [inline src/docs/04_solution_strategy.adoc]: I´d remove model view controller

File: src/docs/04_solution_strategy.adoc [hunk 2]
```diff
@@ -3,21 +3,21 @@
...
+| Test-Driven-Development | We agreed to write the tests first, so this will allow us to work faster when we have to extend the functionality of the application and be always sure if what we are doing is right | The implementation is previously tested, so you can ensure that the code you are writting is right or wrong | It is more tedious to use this type of development
+| Travis  | Automatically build code, test code and provide and inmediate feedback of the success of the changes | It gives you an automatic feedback of your code, so you don't merge branches that are not mergeable | Sometimes it may confuse some commit and set them as wrong when they are fine
+| Coveralls | Hosted analysis tool which provide stadistics about our code coverage | It gives you an stadistical percentage of test coverage of the application |
```
Comment (themrcesi) [inline src/docs/04_solution_strategy.adoc]: Change coveralls to CodeCov

File: src/docs/04_solution_strategy.adoc [hunk 3]
```diff
@@ -3,21 +3,21 @@
...
+| Test-Driven-Development | We agreed to write the tests first, so this will allow us to work faster when we have to extend the functionality of the application and be always sure if what we are doing is right | The implementation is previously tested, so you can ensure that the code you are writting is right or wrong | It is more tedious to use this type of development
+| Travis  | Automatically build code, test code and provide and inmediate feedback of the success of the changes | It gives you an automatic feedback of your code, so you don't merge branches that are not mergeable | Sometimes it may confuse some commit and set them as wrong when they are fine
+| Coveralls | Hosted analysis tool which provide stadistics about our code coverage | It gives you an stadistical percentage of test coverage of the application | 
+| Code Quality | Write quality code and make it understandable, using the readme for external users understand how the application works and provides information | It gives you a feedback of the code quality stadistics with a grade |
```
Comment (themrcesi) [inline src/docs/04_solution_strategy.adoc]: Change code quality to Codacy

**Review by themrcesi — CHANGES_REQUESTED** [review]
> I would remove the MVC pattern since we actually we don´t follow it 100%.

Take a look please.

File: src/docs/04_solution_strategy.adoc (hunk 3 above)
Comment (alvarogarinf) [reply src/docs/04_solution_strategy.adoc]: I will make those changes

**Review by themrcesi — APPROVED** [review]

**Comment by themrcesi** [comment]
Ready to merge? @alvarogarinf

**Pull request merged**